databricks/provider_test.go

```diff
@@ -23,63 +25,71 @@ var testAccProvider *schema.Provider
 var testMWSProvider *schema.Provider
 
 func init() {
-	testAccProvider = Provider("").(*schema.Provider)
-	testMWSProvider = Provider("").(*schema.Provider)
-	cloudEnv := os.Getenv("CLOUD_ENV")
-
-	// If Azure inject sp based auth, this should probably have a different environment variable
-	// But for now best practice on azure is to use SP based auth
-	if cloudEnv == "azure" {
-		var config service.DBApiClientConfig
-		testAccProvider.ConfigureFunc = func(data *schema.ResourceData) (i interface{}, e error) {
-			return providerConfigureAzureClient(data, &config)
-		}
-	}
-
+	testAccProvider = Provider("dev").(*schema.Provider)
+	testMWSProvider = Provider("dev").(*schema.Provider)
 	testAccProviders = map[string]terraform.ResourceProvider{
 		"databricks": testAccProvider,
 	}
 }
 
-// getIntegrationDBAPIClient gets the client given CLOUD_ENV as those env variables get loaded
-func getIntegrationDBAPIClient(t *testing.T) *service.DBApiClient {
-	var config service.DBApiClientConfig
-	config.Token = getAndAssertEnv(t, "DATABRICKS_TOKEN")
-	config.Host = getAndAssertEnv(t, "DATABRICKS_HOST")
-	config.Setup()
+func getMWSClient() *service.DatabricksClient {
+	client := service.DatabricksClient{
+		Host:     os.Getenv("DATABRICKS_MWS_HOST"),
+		Username: os.Getenv("DATABRICKS_USERNAME"),
+		Password: os.Getenv("DATABRICKS_PASSWORD"),
```

Comment:
we should check if the environment variables are set

Reply:
will do in AWS testing batch